js/script.js

```diff
@@ -159,18 +184,21 @@ function showProject() {
   projectCards.appendChild();
 }
 // eslint-disable-next-line no-unused-vars
-function addBlur() {
-  const blur = document.getElementsByClassName('.container');
-  blur.classList.toggle('blur');
-}
+// function addBlur() {
+//   document.body.classList.add('blur');
+// }
+
+// function removeBlur() {
+//   document.body.classList.remove('blur');
+// }
```

Comment:
- [ ] It is considered best practice to avoid keeping commented codes or unused codes in your work base, as it makes it messy. Kindly remove this unused part of your code. :wrench: